### PR TITLE
Filter options

### PR DESCRIPTION
**Pull request by spacephoton**
CleanShot 2021-06-05 at 22 23 51@2x

### CODE REVIEW

**Comment by spacephoton** [comment]
When screen is small

**Review by chaoran-chen — APPROVED** [review]
Looks good to me!

**Comment by chaoran-chen** [comment]
(Pinging #143 - this addresses a part of the issue.)

**Comment by chaoran-chen** [comment]
@TKGZ, the changes made in #157 can then be reverted, I guess?

**Comment by spacephoton** [comment]
> @TKGZ, the changes made in #157 can then be reverted, I guess?

I think so! 

I'm thinking of adding the additional buttons in the same drop down below the fitler, what do you think ?

Or you'd prefer two buttons like your quick sketch?

**Comment by chaoran-chen** [comment]
Do you have an icon in mind that is explanatory enough and covers both filters and other menu items?

**Comment by spacephoton** [comment]
i'll try something, or maybe just add icon should be fine :)

**Comment by spacephoton** [comment]
Added info filter 
CleanShot 2021-06-06 at 13 11 16@2x

@chaoran-chen

**Comment by chaoran-chen** [comment]
One last request: Could you hide the filter button on the acknowledgement/about/login pages?

Works great otherwise. Feel free to merge :)

File: src/Header.tsx [hunk 1]
```diff
@@ -44,6 +47,139 @@ const Header = () => {
...
+                  className={getDropdownButtonClasses('')}
+                  href='https://github.com/cevo-public/cov-spectrum-website'
+                  rel='noopener noreferrer'
+                  target='_blank'
+                >
+                  Github
+                  {/* <AiOutlineGithub
+                    className='hidden md:block fill-current hover:text-gray-500 rounded-xl filter shadow-xl cursor-pointer ml-1 lg:ml-8'
+                    size={'1.5em'}
+                  /> */}
```
Comment (chaoran-chen) [inline src/Header.tsx]: And please remove the commented-out code.

**Pull request merged**